pkgs/by-name/se/seadrive-fuse/package.nix

```
@@ -0,0 +1,62 @@
+{ fetchFromGitHub
+, pkg-config
+, stdenv
+, autoreconfHook
+, lib
+# Package dependencies
+, libsearpc
+, libselinux
+, libuuid
+, pcre
+, libtool
+, libevent
+, sqlite
+, openssl
+, fuse
+, vala
+, intltool
+, jansson
+, curl
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seadrive-fuse";
+  version = "2.0.16";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seadrive-fuse";
+    rev = "v${version}";
+    sha256 = "072sx4wvj3gbslv3hn4sifr28fy812b8aja9d7phl1w4yix9l55z";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+  buildInputs = [
+    libsearpc
+    libselinux
+    libuuid # Satisfies the 'mount' package requirement. Contains 'mount.pc'
+    pcre
+    libtool
+    libevent
+    sqlite
+    openssl.dev
```

```suggestion
    openssl
```

`.dev` should be implied